app/views/profiles/preferences/_locale_form.html.erb

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Can you remove these 2 blank lines at the beginning of the file

Reply:
Fixed in 9f4ef54c2cff211eacd6f2db2111499a09aff0d9